### PR TITLE
User can see activities displayed

### PR DESCRIPTION
**Pull request by EevanR**
[PT LINK](https://www.pivotaltracker.com/story/show/171228982)

As a user
In order to see a finalized trip
I want to be presented with my displayed activities

- display items from back end activities index

Screen Shot 2020-02-12 at 11 33 28 AM

### CODE REVIEW

File: src/components/ActivitiesList.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  const [activities, setActivities] = useState(null);
+  const [gotActivities, setGotActivities] = useState(false);
+
+  const getActivitiesData = async () => {
+    let response = await getActivities(
+      props.trip)
+    if (response.status == 200) {
+      setActivities(response.data[props.activityType])
+      setGotActivities(true)
+    } else {
```
Comment (mantanquzzmo) [inline src/components/ActivitiesList.jsx]: looks redundant

File: src/components/Hotels.jsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
```
Comment (mantanquzzmo) [inline src/components/Hotels.jsx]: remove useEffect

File: src/components/ActivitiesList.jsx (hunk 1 above)
Comment (EevanR) [reply src/components/ActivitiesList.jsx]: took it out

File: src/components/Hotels.jsx (hunk 2 above)
Comment (EevanR) [reply src/components/Hotels.jsx]: took it out

**Review by sundqvistjohan — APPROVED** [review]
LGTM

**Pull request merged**